core/modules/storyviews/classic.js

```diff
@@ -47,16 +47,16 @@ ClassicStoryView.prototype.insert = function(widget) {
 		// Reset the margin once the transition is over
 		setTimeout(function() {
 			$tw.utils.setStyle(targetElement,[
-				{transition: "none"},
 				{marginBottom: ""}
 			]);
+  		targetElement.style.removeProperty('transition');
 		},duration);
 		// Set up the initial position of the element
 		$tw.utils.setStyle(targetElement,[
-			{transition: "none"},
 			{marginBottom: (-currHeight) + "px"},
 			{opacity: "0.0"}
 		]);
+		targetElement.style.removeProperty('transition');
```

Comment:
Double quotes should be used in the core to keep consistency.

```suggestion
		targetElement.style.removeProperty("transition");
```